avocado/core/virt.py

```diff
@@ -16,16 +16,13 @@
 Module to provide classes for Virtual Machines.
 """
 
-import logging
 import time
 from xml.dom import minidom
 
 import libvirt
 
 from . import remoter
 
-LOG = logging.getLogger('avocado.test')
```

Comment:
Well as I mentioned earlier, the LOG is not being used because you removed it in previous commit. I think it should be squashed to the previous commit, but I know it's just a detail so it won't prevent me from merging it, unless there are other serious issues...